Create automatic blobsaving (#85)
cfw-guide/ios.cfw.guide: _pages/en_US/saving-blobs/Automatic blobsaving.md

Please note that this works only for Procursus jailbreak users, meaning unc0ver and base checkra1n (not odysseyra1n) are not supported {: .notice--danger}

## Saving blobs automatically
1. Search for and download "shshd"

> 2. Done! now blobs will automatically be saved when you download shshd and every sunday. The blobs are available at the [TSS saver website](https://tsssaver.1conan.com/v2/), in order to retrieve them, go to "retrieve" and put in your ECID

Reviewer: ```suggestion
2. Done! now blobs will automatically be saved when you download shshd, every time you rejailbreak, and every week after that. The blobs are available at the [TSS saver website](https://tsssaver.1conan.com/v2/), in order to retrieve them, go to "retrieve" and put in your ECID
```
v1.0.1 now saves them every week starting from the time your device booted. rather than specifically on Sundays.

Author: true

## Running shshd manually

> if youve ever wanted to run shshd manually, here is how:

Reviewer: ```suggestion
If you ever want to run shshd manually, here is how:
```

Author: my English so dog water bruh

> 1. SSH into the device or Download NewTerm 2

Reviewer: ```suggestion
1. SSH into the device or download NewTerm 2
```

2. Type ```sudo /usr/sbin/shshd``` (if it asks for a password and you haven't changed it, type alpine)

> 3. It should say "Link to blobs: https://tsssaver.1conan.com/shsh/" then the devices ECID.

Reviewer: ```suggestion
3. It should say "Link to blobs: https://tsssaver.1conan.com/shsh/" then the device's ECID.
```

> Done! If you encounter any issues, join and ask for support in the [R/jb discord server](https://discord.gg/gKAHPtwcGd)

Reviewer: ```suggestion
Done! If you encounter any issues, join and ask for support in the [r/jailbreak Discord server](https://discord.gg/gKAHPtwcGd)
```
